src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockRawlsDAO.scala

```diff
@@ -54,14 +54,40 @@ class MockRawlsDAO  extends RawlsDAO {
     Some(Map("" -> ""))
   )
 
+  private val publishedRawlsWorkspaceWithAttributes = RawlsWorkspace(
+    "id",
+    "attributes",
+    "att",
+    Option(false),
+    "ansingh",
+    "date",
+    Some("date"),
+    Map(AttributeName("default", "a") -> AttributeBoolean(true),
+      AttributeName("default", "b") -> AttributeNumber(1.23),
+      AttributeName("default", "c") -> AttributeString(""),
+      AttributeName("library", "published") -> AttributeBoolean(true),
```

Comment:
it's worth having at least another `library:` attribute, to ensure that calling code finds the right one.